File: Python Code/Daily_Flash_Code_0/Week9/20Mar/52-DailyFlash_Solutions/20-Mar/Java/prog4.java
class Demo {
	public static void main(String[] args) {
		int num = 1;
		char ch = 'A';
		for(int i = 0; i < 4; i++) {
			num = i * 2;
			for(int j = 0; j < 7 - i; j++) {
				if(j < i) {
					System.out.print("\t");
				} else {
					System.out.print(ch);
					System.out.print(num);
					System.out.print("\t");
					ch++;
					num++;
				}
			}
			System.out.println();
		}
	}
}

File: Python Code/Daily_Flash_Code_0/Week9/20Mar/52-DailyFlash_Solutions/20-Mar/Python/prog4.py
ch = 65
for i in range(4):
    num = i * 2;
    for j in range(7 - i):
        if(j < i):
            print("", end = "\t")
        else:
            print(chr(ch),num, end = "\t")
            ch += 1
            num += 1
    print()
